STEERING-COMMITTEE.md

```diff
@@ -1,32 +1,115 @@
-6. Resolve any dispute from the Technical Oversight Committee.
+The Steering Committee’s responsibilities include:
+
+1. Establishing rules of governance for the Istio project, including creation
+and ratification of bylaws for the Steering Committee and Technical Oversight
+Committee.
+1. Establishing and enforcing principles to guide the Istio project.
+1. Fostering an environment for a healthy and happy community of developers,
+contributors, vendors, and users.
+1. Defining, evolving, and defending a
+[Code of Conduct](CONTRIBUTING.md#code-of-conduct).
+1. Advising the Open Usage Commons on issues relating to the Istio trademark and
+logo, as well as related conformance programs.
+1. Setting marketing and advocacy direction for the project; establishing a
+publishing schedule and vetting content, encouraging and assisting community
+members in creating content for conferences, fostering an ecosystem of vendors.
+1. Controlling and delegating access to, and establishing processes regarding,
+project resources/assets, including but not limited to artifact repositories,
+build and test infrastructure, web sites and their domains, blogs, and social
+media accounts.
+1. Providing neutral mediation as appropriate to try to resolve non-technical
+disputes that arise as part of the project.
+
+## Membership and voting
+
+1. A Company shall mean an entity which employs a member, and all other
+entities that control, are controlled by, or are under common control with that
+entity.
+1. The Steering Committee is structured to allow Companies who are most
+invested in the success of the Istio project to participate in business and
+non-technical decision-making.
+1. All members should abide by the project Code of Conduct.
+1. There are two types of seats on the Steering Committee: Contribution Seats
+and Community Seats.
+1. Both Contribution and Community Seats will be appointed beginning in
+August 2020. The appointments for the Contribution and Community seat types will
+expire on staggered dates. After the initial term for both seats, all seats
+will have an annual term.
+1. Contribution Seat terms expire on January 31 and Community Seat terms expire
+on July 31. If necessary, a company holding a Contribution Seat may change the
+appointed individual at any time during the term.
+1. No Company can have more than 5 seats in total on the Steering Committee.
+1. There shall be nine **Contribution Seats**.  Their allocation is determined
+by the approximate effort and expenditure on the Istio project. Each year, when
+Contribution Seat terms begin, the Steering Committee will meet and vote on an
+exact formula and procedure for determining allocation for the following year.
+That procedure will be published to the istio/community repository;
+modifications subsequent to the annual publication of rules shall be considered
+modifications to the Steering Committee Charter.
+    1. The top three Companies contributing to Istio are eligible for
```

Comment:
Noting that we discussed changing this to top three companies are guaranteed a seat, rather than only the top three being eligible.

Reply:
Thanks for the comment. Will adjust (probably to "are guaranteed a seat")